tools/blocksconvert/scanner/scanner.go

```diff
@@ -231,15 +258,33 @@ func (s *Scanner) running(ctx context.Context) error {
 		return allTables[i].start.After(allTables[j].start)
 	})
 
+	for ix := 0; ix < len(allTables); {
+		t := allTables[ix]
+		if s.cfg.PeriodStart.IsSet() && !t.end.IsZero() && t.end.Unix() <= s.cfg.PeriodStart.Unix() {
```

Comment:
In the `PeriodStart` CLI flag description there's:
> Specified date is included in the range. 

If so, shouldn't the check here being `t.end.Unix() < s.cfg.PeriodStart.Unix()`?

Reply:
I actually realised the `t.end` is computed as:
`end:    start.Add(tablesConfig.Period),`

So the `end` is excluded. I would suggest to add a comment to document it. The `if` logic here is fine.

Reply:
> I would suggest to add a comment to document it.

`end` already has this comment:

```
	end    time.Time // Will not be set for non-periodic tables. Exclusive.
```

What do you suggest?

Reply:
All good. Sorry, but I didn't notice that comment.

Reply:
Thanks